docs/UserGuide.md

```diff
@@ -77,73 +70,254 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds a contact to the contact list. A name will be required, and all other information will be optional.
+
+
+Format: `add NAME [-p PHONE_NUMBER] [-e EMAIL] [-s SKILL]...`
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+
+Parameters:
+- `NAME`: Name of the contact. It can be any string except that it cannot start with the hashtag (#) symbol (Reason: To differentiate between indices and names, to be introduced later.), and words cannot start with the hyphen (-) character. (Reason: It will be mixed up with parameter tags.)
+- `PHONE_NUMBER` (optional): Phone number of the contact. Accepts strings with numeric characters only.
+- `EMAIL` (optional): Email of the contact. Accepts any strings without spaces.
+- `SKILL` (zero or multiple allowed): Skill(s) of the person. Accepts any strings, except that words cannot start with the hyphen (-) character. (Reason: It will be mixed up with parameter tags.)
 
 <box type="tip" seamless>
 
-**Tip:** A person can have any number of tags (including 0)
+**Tip:** A person can have any number of skills (including 0)
 </box>
 
+
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+
+- `add John Doe`
+- `add John Doe -p 87654321 -e johndoe@example.com -s Leadership -s C++`
+
+Possible outputs:
```

Comment:
I don't think we need to include possible outputs. Expected outputs should be self-explanatory anyways (so no user guide lookup is expected).